Caltech-IPAC/firefly
# IRSA-2274: export only the original meta

https://jira.ipac.caltech.edu/browse/IRSA-2274
test deploy: https://irsawebdev9.ipac.caltech.edu/irsa-2274/firefly/
Additional changes in firefly_test_data under the same branch name; updated test file.

This change is to preserve the original meta information from the source for later use.
One usage is when exporting.  It should only export the original meta, not any that were added by Firefly during processing.
In the future, we should have an option where the original meta information is displayed in the UI.
This behavior applies to all table, including FITS, IPAC, and VOTABLE.

Design Notes:
TableMeta now contains 2 types of meta; attributes and keywords.
`keywords` is a list of the original meta info, which may contains comments and duplicate keys.
`attributes` is a map containing all meta info, both from the source and added ones during processing.

As developers, you should continue to use `setAttribute` when you want to add meta info into the table.

To test:
- Load table from different sources; IRSA catalog, uploaded FITS table, uploaded VOTable, etc.
- Apply several operations to the table, like sort, filter, show/hide some columns
- Save the table.

There should be only the original meta info in the downloaded table.
